### ISSUE TITLE
Issues with object alignment in material theme

### ISSUE THREAD

**Issue by DSpeichert** [DSpeichert/cloudvcl#54]
![image](https://cloud.githubusercontent.com/assets/1254971/25558752/1edd2a46-2cfb-11e7-8d50-49ba5d8d7b43.png)

1. Icon and text are too close
1. Icon and text are almost overlapping
1. This text is wrapping around for no reason and has large font (isn't it too large?)
1. The buttons have top/bottom margins while the text on the left is aligned to the top of the row, which makes them unaligned together

**Comment by abhishekvalanju** [comment]
![image](https://cloud.githubusercontent.com/assets/4966078/25559644/2cc08204-2d0d-11e7-9465-d74cb5872211.png)

**Comment by abhishekvalanju** [comment]
Fixed all issues except 4. Having problems with the table vertical alignment

**Comment by mitchfeldman** [comment]
Yeah I'm having difficulty with #4 as well

**Comment by mitchfeldman** [comment]
@abhishekvalanju 

I was able to get the course name and buttons to line up (Fig 1) by removing the div that contained the buttons (Fig 2) but now we need to figure out how to get the buttons to line up on the far side with CSS. 

Fig 1.
![image](https://cloud.githubusercontent.com/assets/1210738/25633381/df6f523e-2f44-11e7-8858-5de66d0da3c2.png)

Fig 2.
![image](https://cloud.githubusercontent.com/assets/1210738/25633443/0ff780d4-2f45-11e7-9798-442cb16db70c.png)

**Comment by DSpeichert** [comment]
1 & 2 do not seem to be an issue anymore, magically solved itself.
To fix number 4 I suggest changing the strategy. Instead of changing the amount of space that buttons take, let's try to vertically center-align the text in all table cells so they match buttons. @abhishekvalanju  @mitchfeldman

**Comment by mitchfeldman** [comment]
Yeah I tried that and basically everything I saw online. Getting one button over and lined up is easy but as soon as the other one gets there shit goes awry..

**Comment by mitchfeldman** [comment]
http://stackoverflow.com/questions/43857790/bootstrap-material-design-table-alignment-issues

^ created a post for number 4

**Comment by abhishekvalanju** [comment]
@mitchfeldman, have you tried the solution that is suggested in the stackoverflow above?

**Comment by mitchfeldman** [comment]
Yeah, didn't work. Not sure why he suggested centering it cause we want vertical alignment.. but trying vertical alignment moves it down like 5px. I think something is wrong with the HTML structure of the tables rather than CSS.
